packages/app-content-pages/src/screens/Publications/components/Publication/Publication.js

```diff
@@ -1,30 +1,8 @@
-import { Media, ZooniverseLogo } from '@zooniverse/react-components'
 import { Anchor, Box, Text } from 'grommet'
 import { string } from 'prop-types'
-import styled from 'styled-components'
-
-const StyledBox = styled(Box)`
-  border-radius: 5px;
-  overflow: hidden;
-`
-
-const Placeholder = ({ displayString }) => {
```

Comment:
Optionally, with this moving to Project.js maybe the related publications-publication--placeholders story can be removed and a placeholder related story could be added to the Project.stories.